android/src/main/java/com/brentvatne/exoplayer/PictureInPictureUtil.kt

```diff
@@ -0,0 +1,79 @@
+package com.brentvatne.exoplayer
+
+import android.annotation.SuppressLint
+import android.app.AppOpsManager
+import android.app.PictureInPictureParams
+import android.app.RemoteAction
+import android.content.pm.PackageManager
+import android.graphics.drawable.Icon
+import android.os.Build
+import android.os.Process
+import androidx.annotation.ChecksSdkIntAtLeast
+import androidx.annotation.RequiresApi
+import androidx.core.app.AppOpsManagerCompat
+import com.brentvatne.receiver.PictureInPictureReceiver
+import com.facebook.react.uimanager.ThemedReactContext
+
+class PictureInPictureUtil {
+    companion object {
+        fun enterPictureInPictureMode(context: ThemedReactContext, pictureInPictureParams: PictureInPictureParams?) {
+            if (!isSupportPictureInPicture(context)) return
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O && pictureInPictureParams != null) {
+                context.currentActivity?.enterPictureInPictureMode(pictureInPictureParams)
+            } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+                context.currentActivity?.enterPictureInPictureMode()
+            }
+        }
+
+        fun updatePictureInPictureActions(context: ThemedReactContext, pictureInPictureParams: PictureInPictureParams) {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                context.currentActivity?.setPictureInPictureParams(pictureInPictureParams)
+            }
+        }
```

Comment:
So this is crashing, if we don't add `android:supportsPictureInPicture="true"` to activity manifest

Reply:
@KrzysztofMoch
Thanks. it's my mistake🙀
I fixed with [2872689](https://github.com/react-native-video/react-native-video/pull/3385/commits/28726891122587be2b13cb0780c9a7e037bf2417)